src/wcm/_download.py

```diff
@@ -136,7 +136,12 @@ def download(component_dir, profile=None, download_path=None):
 
         # makes the YAML file
         stream = open(os.path.join(path, "wings-component.yaml"), 'w+')
-        yaml.dump(yaml_data, stream, sort_keys=False)
+        
+        try:
+            yaml.dump(yaml_data, stream, sort_keys=False)
+        except TypeError:
+            yaml.dump(yaml_data, stream)
+            logger.warning("yaml dumb received type error. Trying again without sort_keys")
```

Comment:
Not needed as we can require a newer version of PyYAML.

Reply:
Might come in handy if someone, for some reason, reverts pyyaml after installin wcm. I figured it cant hurt

Reply:
Since Chris fixed this, I approved it.